taskcluster/test/params/large-lt-en.yml

```diff
@@ -163,4 +163,9 @@ training_config:
   target-stage: all
   taskcluster:
     split-chunks: 10
+    worker-classes:
+      train-backwards: gcp-standard
```

Comment:
not sure if it's important here but there's also finetune-student

Reply:
It won't really make a difference to the consumers of this, but I may as well add it, as I meant this to be production-like.